medical-portal/src/UI/src/app/dashboard/dashboard.component.html

```diff
@@ -182,7 +182,7 @@ <h1>Portal Dashboard</h1>
                       }
 
                       @if (
-                        searchedCase.status === DMERStatusEnum.RequiredUnclaimed
+                        searchedCase.status === DMERStatusEnum.RequiredUnclaimed && searchedCase.isOwner == true
```

Comment:
in the future, this will also need to check if the claimed user is in the network, as mentioned by Shiv today